Firestore/core/src/firebase/firestore/local/local_serializer.cc

```diff
@@ -32,12 +34,13 @@ namespace firebase {
 namespace firestore {
 namespace local {
 
-using firebase::firestore::model::ObjectValue;
-using firebase::firestore::model::SnapshotVersion;
-using firebase::firestore::nanopb::Reader;
-using firebase::firestore::nanopb::Tag;
-using firebase::firestore::nanopb::Writer;
-using firebase::firestore::util::Status;
+using core::Query;
```

Comment:
Might be worth to add to go/firestore-cpp-style as well i.e. in favor of relative namespace than full-qualified namespace. I am fine either way but not mixed. Could you go over the Firestore C++ code base and fix the others, if any, as well? (maybe do a git grep using to audit.)